common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1633,7 +1633,12 @@ def rescore(self, only_if_higher=False):
         # rescoring should have no effect on attempts, so don't
         # need to increment here, or mark done.  Just save.
         self.set_state_from_lcp()
-        self.set_score(calculated_score)
+        old_percentage = orig_score.raw_earned / orig_score.raw_possible
```

Comment:
Can you check for a better comparison?
https://github.com/edx/edx-platform/blob/master/openedx/core/lib/grade_utils.py#L14-L14